docs/source/mobile_optimizer.rst

```diff
@@ -0,0 +1,26 @@
+torch.utils.mobile_optimizer
+.. warning::
+
+This API is in beta and may change in the near future.
+
+Torch mobile supports :mod:`torch.mobile_optimizer.optimize_for_mobile` utility to run a list of optimization pass with modules in eval mode.
+The method takes the following parameters:
+
+1. A TorchScript object;
+2. A set of blacklisted optimization;
+3. A list of preserved methods.
+
+
+.. note::
+By default, if optimization blacklist is None or empty, “optimize_for_mobile” will run the following optimizations:
```

Comment:
Suggestion: Use either italics or code syntax for optimize_for_mobile of quotation marks.